lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
@ormsbee The original thinking was that certificates would not be used in our use cases. The presumption was that edX would not offer certificates for POCs, and that POC coaches would have other ways to recognize student achievement. 

However, on further discussions with the course team from 15.390.1x, I'm less sure this is the case. @pwilkins have you heard anything further?

@ichuang, @ferdi can you weigh in here?

On Jan 26, 2015, at 12:02 PM, David Ormsbee <notifications@github.com> wrote:

> In lms/djangoapps/courseware/grades.py:
> 
> > @@ -267,6 +268,8 @@ def create_module(descriptor):
> >  
> >          totaled_scores[section_format] = format_scores
> >  
> > +    # Grading policy might be overriden by a POC, need to reset it
> > +    course.set_grading_policy(course.grading_policy)
> Relating to this -- how would this feature deal with the question of certificate generation (given the policy differences)?
> 
> —
> Reply to this email directly or view it on GitHub.
>